### ISSUE TITLE
Installation failed on Version 18.03.0-ce-rc1-mac54

### ISSUE THREAD

**Issue by artemkaint** [mal/docker-for-mac-host-bridge#7]
Move original hyperkit
mv: rename hyperkit to hyperkit.real: No such file or directory

ls -la /Applications/Docker.app/Contents/Resources/bin/

```
-rwxr-xr-x   1 artemkaint  admin   4876320 Feb 27 18:05 com.docker.hyperkit
-rwxr-xr-x   1 artemkaint  admin  20042784 Feb 27 18:05 com.docker.vpnkit
-rwxr-xr-x   1 artemkaint  admin  38267040 Feb 27 18:05 docker
-rwxr-xr-x   1 artemkaint  admin   6338528 Feb 27 18:05 docker-compose
-rwxr-xr-x   1 artemkaint  admin   1633424 Feb 27 18:05 docker-credential-osxkeychain
-rwxr-xr-x   1 artemkaint  admin   4768000 Feb 27 18:05 docker-diagnose
-rwxr-xr-x   1 artemkaint  admin  31535504 Feb 27 18:05 docker-machine
-rwxr-xr-x   1 artemkaint  admin  67727776 Feb 27 18:05 kubectl
-rwxr-xr-x   1 artemkaint  admin   8453936 Feb 27 18:05 notary

```

**Comment by mal** [comment]
Hi @artemkaint, thanks for the heads up!

I've just pushed a [`beta`](https://github.com/mal/docker-for-mac-host-bridge/tree/beta) branch which solves the issue you're encountering, but I'm holding off on a proper release while I investigate some networking issues that seem to have arisen (sometimes you can connect to the local network, sometimes to the internet), but please feel free to try it out and report back any issues you find.

I hope to have something more concrete tomorrow, but depending on how docker has changed, I can't promise any results at this stage.

**Comment by mal** [comment]
Issue I've been seeing matches the description of the issue in #5, so given it's pre-existing I'll proceed with shipping this.

**Comment by mal** [comment]
Shipped in [`1.2.0`](https://github.com/mal/docker-for-mac-host-bridge/tree/v1.2.0).
